Server/API/rooms/postRoom.go

```diff
@@ -0,0 +1,34 @@
+package rooms
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	roomsScripts "ishtaloo.io/DB/Scripts/rooms"
+	entities "ishtaloo.io/Entities"
+	cookieUtils "ishtaloo.io/Utils/cookie"
+)
+
+func postRoom(c echo.Context) (err error) {
+	r := new(entities.Room)
+	if err = c.Bind(r); err != nil {
+		return err
+	}
+	u, _ := cookieUtils.GetUserFromCookie(c)
+	users := make([]entities.User, 10)
```

Comment:
10?

Reply:
Just a number I picked for maximum number of particpants

Reply:
Make it a constant for now